desk/app/chat.hoon

```diff
@@ -256,13 +256,25 @@
   ++  create
     |=  req=create:c
     ^+  cor
+    |^
+    ?.  can-nest
+      %.  cor
+      (slog 'Create failed: group not found' ~)
```

Comment:
naughty! please crash here

Reply:
```
~_  leaf+"Create failed: check group permissions"
?>  can-nest
```

Now in both diary and chat. Did not know what heap was until like 2 days ago lol. Will resolve.